.github/workflows/ci.yml

```
@@ -94,16 +88,18 @@ jobs:
         shell: bash
   gradle-emulator-test:
     runs-on: ubuntu-latest
-    needs: [build, gradle-wrapper-validation]
+    needs: [build]
     timeout-minutes: 20
     steps:
       - name: Check out repository code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Install Java 17
         uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
           java-version: '17'
+      - name: Gradle wrapper validation
+        uses: gradle/wrapper-validation-action@v1
```

```suggestion
        uses: gradle/wrapper-validation-action@v2
```

https://github.com/gradle/wrapper-validation-action/releases/tag/v2.0.0-rc.1 released few minutes ago.

---

Up to you, it's only [minor and patch bumps](https://github.com/gradle/wrapper-validation-action/compare/v1.1.0...v2.0.0-rc.1#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) apart from the Node 20 change. 
I wouldn't expect issues from this action, Gradle's [RCs](https://docs.gradle.org/current/userguide/feature_lifecycle.html#eol_support) are pretty stable usually and final release won't require any changes.

Some examples where it's working fine:
https://github.com/ReactiveCircus/android-emulator-runner/pull/374
https://github.com/junit-pioneer/junit-pioneer/pull/804